MAUI/DataGrid/Column-types.md

```diff
@@ -337,6 +337,56 @@ dataGrid.Columns.Add( new DataGridTextColumn()
 {% endhighlight %}
 {% endtabs %}
 
+### Formatting DataGridTextColumn with different culture
+
+To apply different CultureInfo for DataGridColumns, use the `DataGridColumn.CultureInfo` property. Assign format of the string to this property. Based on the type of the property the column is associated to format the value. You can use different StringFormats to customize values displayed in the record cells.
+
+To apply different cultures for the GridColumns, follow the code example:
```

Comment:
To apply different cultures for the `DataGridColumns`, follow the code example:

Reply:
Changed